src/loaders/common/loader_helper.py

```diff
@@ -77,3 +84,53 @@ def init_genome_atrri_doc(kbase_collection, load_version, genome_id):
     }
 
     return doc
+
+
+def get_token(token_filename):
+    """
+    Get token if a filename is provided at home dir.
+    """
+    try:
+        home_dir = os.path.expanduser("~")
```

Comment:
Don't assume that the file is in the user's home dir. Go ahead and `expanduser` on `token_filename` though

Reply:
Updated!

Reply:
:+1: